packages/sdk/src/chains/Substrate/__test__/deposit.test.ts

```diff
@@ -1,7 +1,7 @@
 import { ApiPromise } from '@polkadot/api';
 import { XcmMultiAssetIdType } from '../types';
 
-import * as Utils from '../utils/depositFns.js';
+import * as Utils from '../utils/depositFns';
```

Comment:
why this change reference to the `.js` file, but it is present above?

Reply:
For some reason when using the automatic `update import` functionality, VSCode sometimes appends the `.js` suffix, but this is technically not required. Hence I am removing it.